aurelia-app-identity-server-aspnetcore/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:5000/",
```

Comment:
Good call to set this in launchSettings.json for consistency's sake. 

In the [Program.cs][0] file, we used port `9000`, because that matches the [`open-id-connect-configuration.ts`][1] files. If you change this to `9000` instead of `5000`, then we'll merge it. The port is important, because we register only `localhost:9000` as the redirect_uri at the OpenID Connect Provider.

[0]: https://github.com/shaunluttin/aurelia-open-id-connect-demos/blob/master/aurelia-app-aspnet-core/Program.cs#L22

[1]: https://github.com/shaunluttin/aurelia-open-id-connect-demos/blob/master/aurelia-app-aspnet-core/ClientApp/open-id-connect-configuration-auth0.ts#L4

Reply:
For my understanding, why the `.UseUrls("http://localhost:9000")` in [Program.cs](https://github.com/shaunluttin/aurelia-open-id-connect-demos/blob/master/aurelia-app-aspnet-core/Program.cs#L22)